lib/Backends/CPU/CPUDeviceManager.cpp

```diff
@@ -89,9 +91,12 @@ void CPUDeviceManager::addNetworkImpl(const Module *module,
                   .str()));
       return;
     }
+
+    allFunctionsMemoryBytes +=
```

Comment:
Isn't the runtimeBundle per function? Why do we need the allFunctionsMemoryBytes?

Reply:
I may not be up to date on how `addNetwork` is supposed to work, but I thought every function in the provided map `functions` has to be added to the device, so you need to add up the sizes of the constants for each function.

Reply:
You're right, I thought it was outside the loop, looks good.